cluster/ci/monitoring/Makefile

```diff
@@ -33,6 +34,9 @@ prow-monitoring-deploy: ci-secret-deploy
 	#oc create -n prow-monitoring configmap grafana-dashboard-deck --from-file=./deck.json --dry-run -o yaml | oc apply -f -
 	#oc create -n prow-monitoring configmap grafana-dashboard-build-cop --from-file=./build_cop.json --dry-run -o yaml | oc apply -f -
 	#oc create -n prow-monitoring configmap grafana-dashboard-prow --from-file=./prow.json --dry-run -o yaml | oc apply -f -
+
+	# Fill prometheus-k8s basicAuthPassword in datasources file.
+	sed -i -e "s|CHANGE_ME|$(prometheus-k8s_token)|g" ./datasources.yaml
```

Comment:
Would `sed` line has to be ahead of `oc create -n prow-monitoring configmap grafana-datasources --from-file=./datasources.yaml --dry-run -o yaml | oc apply -f -` above?

NIP: I would also suggest to recover the change (not leaving any secret even in the CI pod) after creating the configMap ... We could do this by a `git checkout ./datasources.yaml`